tests/functional/common.py

```diff
@@ -0,0 +1,98 @@
+from typing import Callable, Iterable
+
+import pytest
+from duckdb import DuckDBPyConnection
+from ibis.expr.types.relations import Table
+
+from tests.verification import verify_equals
+
+
+def check_subtrait_function_names(
+    substrait_plans: dict,
```

Comment:
Is this multiple plans or a single plan?  It looks like a single plan the way you interact with it.

Reply:
Oh yeah, it's just a single plan.  I'll make it singular.